iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/dataregion/compaction/execute/utils/executor/batch/BatchedReadChunkAlignedSeriesCompactionExecutor.java

```diff
@@ -328,7 +328,7 @@ public FollowingBatchedReadChunkAlignedSeriesGroupCompactionExecutor(
     @Override
     protected ChunkLoader getChunkLoader(TsFileSequenceReader reader, ChunkMetadata chunkMetadata)
         throws IOException {
-      if (chunkMetadata != null && chunkMetadata.getMeasurementUid().isEmpty()) {
+      if (chunkMetadata != null && AlignedSeriesBatchCompactionUtils.isTimeChunk(chunkMetadata)) {
```

Comment:
Also remove the nullability check here?

Reply:
The nullability check here is necessary because it is checked in super.getChunkLoader